app/src/main/java/org/simple/clinic/editpatient/EditPatientScreenKey.kt

```diff
@@ -27,5 +27,5 @@ data class EditPatientScreenKey(
   @IgnoredOnParcel
   override val analyticsName = "Edit Patient"
```

Comment:
Should the analytics name for the new screen be different?

Reply:
@vinaysshenoy is there a reason why we should make that distinction?

Reply:
I thought we might want to since it's a new screen, but then I realised that from the POV of the user, the screen hasn't changed. We already have a separate event for the architecture migration, so this is unnecessary. Closing the comment.